src/index.tsx

```diff
@@ -195,21 +195,28 @@ export default function AxeMode({ children, disabled }: AxeModeProps) {
     }
 
     if (childrenRef.current) {
-      if (idleId) {
+      if (idleId && 'cancelIdleCallback' in window) {
         // requestIdleCallback has no types:
         // https://github.com/microsoft/TypeScript/issues/21309
         // @ts-ignore
         cancelIdleCallback(idleId);
         setIdleId(null);
       }
 
-      // requestIdleCallback has no types:
-      // https://github.com/microsoft/TypeScript/issues/21309
-      // @ts-ignore
-      const id = requestIdleCallback(() => {
+      function getViolations() {
```

Comment:
In strict mode the compiler will get angry about this function definition being inside of the `if` block. I'd move it outside the block but still in the effect.